app/src/main/java/com/android/ubclaunchpad/driver/UI/BaseMenuActivity.java

```diff
@@ -52,10 +53,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     public void signOut() {
```

Comment:
Please add java doc comments so everyone else knows that this sign out clears all the activities in the back stack and finishes the activity off just to be clear